### ISSUE TITLE
definitly typed CI failes because of dtslint

### ISSUE THREAD

**Issue by Flarna** [Microsoft/dtslint#186]
it seems dtslint has problems with versioned type definitions. jsdom issues following error if I execut it local:
```
Error in jsdom
Error: c:/Workspaces/GithubForks/DefinitelyTyped/types/jsdom/ts3.1/index.d.ts:1:1
ERROR: 1:1  expect  Program source files differ between TypeScript versions. This may be a dtslint bug.
Expected to find a file 'c:/Workspaces/GithubForks/DefinitelyTyped/types/jsdom/ts3.1/index.d.ts' present in 3.3.0-dev.20190124, but did not find it in ts@3.0.
```
In CI for node we get 
```
Error in jsdom
Error: Expected compilerOptions["baseUrl"] === "../"
    at Object.<anonymous> (/home/travis/build/DefinitelyTyped/DefinitelyTyped/node_modules/dtslint/bin/checks.js:72:27)
    at Generator.next (<anonymous>)
    at fulfilled (/home/travis/build/DefinitelyTyped/DefinitelyTyped/node_modules/dtslint/bin/checks.js:4:58)
    at <anonymous>
```

see also https://github.com/DefinitelyTyped/DefinitelyTyped/issues/32444

**Comment by Flarna** [comment]
seems the same happens now with google-cloud__tasks introduces via https://github.com/DefinitelyTyped/DefinitelyTyped/pull/32051, error in node CI see https://github.com/DefinitelyTyped/DefinitelyTyped/pull/32552:

```
Error in google-cloud__tasks
Error: Expected compilerOptions["baseUrl"] === "../"
    at Object.<anonymous> (/home/travis/build/DefinitelyTyped/DefinitelyTyped/node_modules/dtslint/bin/checks.js:72:27)
    at Generator.next (<anonymous>)
    at fulfilled (/home/travis/build/DefinitelyTyped/DefinitelyTyped/node_modules/dtslint/bin/checks.js:4:58)
    at <anonymous>
```

**Comment by sandersn** [comment]
These don't repro locally for me. Can you repro them outside Travis? My best guess right now is a stale CI environment.

**Comment by Flarna** [comment]
Yes, I can reproduce it locally (using windows). Did a fresh rebase to latest on master and cleaned node_modules folders.

**Comment by sandersn** [comment]
What command are you using? I see this with `npm run test` but not `npm run lint jsdom`

**Comment by Flarna** [comment]
I use `npm run test` as this is what CI does.

**Comment by sandersn** [comment]
Fixed in 4e914260b6bcc081da05c9fe14d9938e6c4f72d3
